pkg/setup/setup.go

```diff
@@ -358,3 +377,41 @@ Do you trust it?`
 		fingerprintBuf.String(),
 	))
 }
+
+// installBundledPlugin installs the default core plugin bundled with the wrapper CLI
+//
+// This will occur for two primary reasons:
+// 1. The cluster and the computer running setup are airgapped. By default the resources describing
+// where the core plugins are point to S3 which will be unreachable in an airgapped environment.
+// 2. The user running setup does not have permission to access Cosmos (dcos:adminrouter:package).
+//
+// Though a user could install the core plugin manually, this will prevent usability regressions until
+// other features of DC/OS are there to allow the normal path to handle all cases.
+func (s *Setup) installBundledPlugin() error {
+	pluginData, err := Asset(path.Join(runtime.GOOS, bundledCoreFilename))
+	if err != nil {
+		return err
+	}
+
+	// Write out the data into a temp directory so that it's in the real filesystem for buildPlugin
+	bundleTempDir, err := afero.TempDir(s.fs, os.TempDir(), "dcos-default-plugin")
+	if err != nil {
+		return err
+	}
+	bundleZipPath := path.Join(bundleTempDir, "dcos-core-cli.zip")
+	err = afero.WriteFile(s.fs, bundleZipPath, pluginData, 0644)
+	if err != nil {
+		return err
+	}
```

Comment:
Having a temp dir is not strictly needed, It'd be simpler to write to a temp file directly here :
``` go
f, err := afero.TempFile(s.fs, os.TempDir(), "dcos-core-cli")
if err != nil {
	return err
}
_, err = f.Write(ca)
if err != nil {
	return err
}
```